### PR TITLE
Fix miscellaneous LED grouping

### PR DESCRIPTION
**Pull request by jinuthomas**
added unique groups for each miscellaneous led

Signed-off-by: Jinu Joy Thomas <jinu.joy.thomas@in.ibm.com>

### CODE REVIEW

**Comment by jinuthomas** [comment]
fixes SW546458 partially will also need https://github.ibm.com/openbmc/openbmc/pull/2088 to go in together

File: configs/ibm,everest/led-group-config.json [hunk 1]
```diff
@@ -9737,7 +9737,813 @@
...
+               "Name" : "virtual_enc_id",
+               "Action" : "On",
+               "DutyOn" : 50,
+               "Period" : 0,
+               "Priority" : "Blink"
+            }
+         ]
+      },
+      {
+         "group" : "c4_cxp_top_identify",
```
Comment (santoshpuranik) [inline configs/ibm,everest/led-group-config.json:10250]: These already seem to be there as cablecard_* groups

File: configs/ibm,rainier-1s4u/led-group-config.json [hunk 2]
```diff
@@ -3457,7 +3457,1567 @@
...
+               "Name" : "virtual_enc_id",
+               "Action" : "On",
+               "DutyOn" : 50,
+               "Period" : 0,
+               "Priority" : "Blink"
+            }
+         ]
+      },
+      {
+         "group" : "c0_cxp_top_identify",
```
Comment (santoshpuranik) [inline configs/ibm,rainier-1s4u/led-group-config.json:4214]: Same with these - I think the cable card CXP ports have a physical LED, so I think we should not need these.

**Review by santoshpuranik — CHANGES_REQUESTED** [review]
Have some queries about the CXP ports and we'll need to propagate any changes to group names (and maybe even additions) to https://github.com/ibm-openbmc/pldm/tree/1020/oem/ibm/configurations/pdr

**Review by santoshpuranik — APPROVED** [review]

**Pull request merged**